### DISCUSSION TITLE
How increase message queue rate

### DISCUSSION THREAD

**Discussion by jayaraj** [emqx/emqx#11664] [General]
Hi,
 While stress testing  with EMQX 5.1.6 My client is getting disconnected logging 

`2023-09-23T16:15:34.577189+00:00 [error] supervisor: 'esockd_connection_sup - <0.2807.0>', errorContext: connection_shutdown, reason: #{max => 1000,reason => message_queue_too_long,value => 1276}, offender: [{pid,<0.4357.0>},{name,connection},{mfargs,{emqx_connection,start_link,[#{enable_authn => true,limiter => #{client => #{messages => #{burst => 0,divisible => true,failure_strategy => force,initial => 0,low_watermark => 0,max_retry_time => 3600000,rate => 5.0e3}}},listener => {tcp,default},zone => default}]}}]`

and my env setup is 

```
 environment:
      - EMQX_NAME=emqx
      - EMQX_WAIT_TIME=20
      - EMQX_ALLOW_ANONYMOUS=true
      - EMQX_NODE__PROCESS_LIMIT=2097152
      - EMQX_NODE__MAX_PORTS=2097152
      - EMQX_LISTENER__TCP__EXTERNAL__ACCEPTORS=64
      - EMQX_LISTENER__TCP__EXTERNAL__MAX_CONNECTIONS=1024000
      - EMQX_ZONE__INTERNAL__MAX_MQUEUE_LEN=0
      - EMQX_ZONE__EXTERNAL__MAX_MQUEUE_LEN=0
      - EMQX_ZONES__DEFAULT__MQTT__MAX_MQUEUE_LEN=1000000
      - EMQX_ZONES__INTERNAL__MQTT__MAX_MQUEUE_LEN=1000000
      - EMQX_ZONES__EXTERNAL__MQTT__MAX_MQUEUE_LEN=1000000
      - EMXQ_SESSION__MAX_MQUEUE_LEN=100000
      - EMQX_LOADED_PLUGINS="emqx_management,emqx_recon,emqx_retainer,emqx_dashboard,emqx_web_hook"
```

Can any one help me configure EMQX to handle unlimited message/s
I was able to connect more than 100K clients.

**Comment by sstrigler** [comment]
Hey,

please try with 
```
EMQX_ZONES__DEFAULT__MQTT__MAX_MQUEUE_LEN=infinity
```
